app/controllers/TaskController.scala

```diff
@@ -420,17 +420,31 @@ class TaskController @Inject()(annotationDAO: AnnotationDAO,
             dataSet._id,
             description = tuple._1._1.description
         ))
+      warnings <- warnIfTeamHasNoAccess(requestedTasks.map(_._1), dataSet)
       zippedTasksAndAnnotations = taskObjects zip annotationBases
       taskJsons = zippedTasksAndAnnotations.map(tuple => taskToJsonFoxed(tuple._1, tuple._2))
       result <- {
         val taskJsonFuture: Future[List[Box[JsObject]]] = Fox.sequence(taskJsons)
         taskJsonFuture.map { taskJsonBoxes =>
-          bulk2StatusJson(taskJsonBoxes)
+          Json.obj("tasks" -> bulk2StatusJson(taskJsonBoxes), "warnings" -> warnings)
         }
       }
     } yield Ok(Json.toJson(result))
   }
 
+  private def warnIfTeamHasNoAccess(requestedTasks: List[TaskParameters], dataSet: DataSet)(
+      implicit ctx: DBAccessContext): Fox[List[String]] = {
+    val projectNames = requestedTasks.map(_.projectName).distinct
+    for {
+      projects: List[Project] <- Fox.serialCombined(projectNames)(projectDAO.findOneByName(_))
+      dataSetTeams <- teamDAO.findAllForDataSet(dataSet._id)
+      noAccessTeamIds = projects.map(_._team).filter(projectTeam => !dataSetTeams.map(_._id).contains(projectTeam))
```

Comment:
```suggestion
      noAccessTeamIds = projects.map(_._team).diff(dataSetTeams.map(_._id)))
```

I feel like a diff would be more concise here :slightly_smiling_face: